### ISSUE TITLE
TODO: AdvancedBan import option

### ISSUE THREAD

**Issue by ruany** [ruany/LiteBans#17]
https://www.spigotmc.org/resources/advancedban.8695/

This would be high priority if there was more demand, but I've only had a few requests so far.
Importing was originally intended for plugins that didn't have UUID support at the time, like MaxBans and BanManagerv4 (the original MaxBans still doesn't have actual UUID support despite claiming to have it).

AdvancedBan was released after LiteBans was first released, unlike all of the other plugins which can currently be imported from.

**Comment by eUipKh** [comment]
I would like to bought this plugin but to do that i really need this.

**Comment by eUipKh** [comment]
Any updates?

**Comment by ruany** [comment]
Not quite, still a lack of demand for this. It's being considered but I can't promise anything as I have other things to work on.
